### ISSUE TITLE
hot reloading

### ISSUE THREAD

**Issue by 8balloon** [8balloon/eigenstate#21]
```js
require.context('./', true, /.*\.jsx?$/)

<HotProvider stateDefPath="./stateDef" viewComponentPath="./View" onAction={logVerbosely} />
```

**Comment by 8balloon** [comment]
#20 makes it so you can implement this yourself using Webpack. I'm starting to think that this isn't worth it for that reason. Also I kind of hate hot reloading, an opinion which might change if I think of a better way to merge state than was implemented in #20.

**Comment by 8balloon** [comment]
Closing, since you can implement this yourself and it would create a dependency on webpack. It would be better to create an example / boilerplate that demonstrates this implementation, along with React Router hookup, and hard stuff like that.
